demo/src/main/java/com/hazelcast/demo/AllTest.java

```diff
@@ -25,7 +25,7 @@
 import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.core.IMap;
 import com.hazelcast.internal.diagnostics.HealthMonitorLevel;
```

Comment:
Not sure if there is any other way, but do we really want our examples to depend on `impl`/`internal` packages?